verilog/parser/BUILD

```diff
@@ -225,3 +225,25 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+genrule(
+    name = "verilog_token_enum_foreach_gen",
+    srcs = ["verilog_token_enum.h"],
+    outs = ["verilog_token_enum_foreach.inc"],
+    cmd = "sed -n -e '1,/^\\s*enum verilog_tokentype/d' " +
+          " -e '/^\\s*\\w\\+/ s/^\\s*\\(\\w\\+\\).*$$/CONSIDER(\\1)/p' " +
+          " -e '/^\\s*};/q' $< > $@",
```

Comment:
I'm a little concerned about the use of some of the Perl-compatible/GNU sed extensions like `\w` and `\s`.  If we find problems in testing, we may have to rewrite these in a more compatible manner.

Reply:
code removed